owtf/webapp/src/containers/Report/Table.js

```diff
@@ -114,6 +114,18 @@ export class DataTable extends React.Component {
     const code = obj["plugin_code"];
     const user_notes = obj["user_notes"];
     const editorShow = this.state.editorShow;
+    let resourceList = [];
+    try {
+      output.map(singleOutput => {
+        if (singleOutput.hasOwnProperty('output')) {
+          if (singleOutput.output.hasOwnProperty('ResourceList')) {
+            resourceList = singleOutput.output.ResourceList
+          }
+        }
+      })
```

Comment:
store the **ResourceListName** also in a variable

Reply:
Thank you for reviewing. Will make the changes.